contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,652 @@
+            <td className="text-center"><span className="text-red text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Build your own apps and contribute code</td>
+        </tr>
+        <tr>
+            <td><strong>Apps/Integrations</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Push and pull data to other destinations</td>
+        </tr>
+        <tr>
+            <td><strong>Open source</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Build your own apps and contribute code</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **[Session replays](/session-replay):** Session replays in PostHog, otherwise known as session recording, enable you to capture video recordings of a user experience. Additionally, in PostHog, you can view built-in console logs, and network performance. These features are especially useful for diagnosing bug reports, or building user empathy. Pendo does not offer any session replay feature, forcing users to turn to tools such as HotJar. 
+
+- **[Feature flags](/feature-flags):** PostHog offers multivariate feature flags which support JSON payloads, enabling you to push real-time changes to a product without needing to redeploy. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more. Pendo, on the other hand, does not support any feature flag functionality and users will need to adopt an alternative feature flagging tool.
+
+- **[A/B testing](/ab-testing):** In PostHog, you can use the experimentation suite to create multivariate tests within your product, such as showing some users a different page layout to others. Over time, you can build an understanding of which page performs better, correlate results with other events and deploy a final version. You can’t do any of this in Pendo. 
+
+- **[Open source](/product-os):** PostHog is open source. You can [view our source code](httpw://github.com/posthog), contribute changes, engage with the team in [our open source community](/slack), [view our roadmap](/roadmap) and create your own [apps](/apps) or features. Pendo is not open source and, as a result, you can only suggest new features to the support team.
+
+### Product Analytics
+
+PostHog and Pendo both offer similar product analytics features at a surface level, but differ significantly when compared in greater depth. This is because PostHog and Pendo are ultimately built for different types of users. While PostHog is designed with engineers and technical users in mind, Pendo is intended more for marketers and UX designers. 
```

Comment:
I don't feel like the comparison can hold up the "greater depth" comment atm. Outside of some specific features, what about PostHog provides greater depth?